csharp/GildedRoseTest.cs

```diff
@@ -12,7 +12,130 @@ public void foo()
             IList<Item> Items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
             GildedRose app = new GildedRose(Items);
             app.UpdateQuality();
-            Assert.AreEqual("fixme", Items[0].Name);
+            Assert.AreEqual("foo", Items[0].Name);
+        }
+
+        [Test]
+        public void AfterSellByDate_QualityDegradesTwiceAsFast()
+        {
+            IList<Item> items = new List<Item> { new Item { Name = "Big Mac", SellIn = 5, Quality = 50 } };
+            GildedRose app = new GildedRose(items);
+
+            for (int i = 1; i <= 10; ++i)
```

Comment:
Why do you need to call this 10 times? If the same thing's happening in all 10 cases, then we can only call it once. If it's not the same thing happening, we need 2 or more tests

Reply:
This applies to all tests, really